tests/integration/istio_io/mtls_test.go

```diff
@@ -119,20 +102,21 @@ func validateInitialDestinationRules(output string, err error) error {
 //https://istio.io/docs/tasks/security/mtls-migration/
 //https://github.com/istio/istio.io/blob/release-1.2/content/docs/tasks/security/mtls-migration/index.md
 func TestMTLS(t *testing.T) {
-	//Test
-	examples.New(t, "mtls-migration").
-		RunScript("create-ns-foo-bar-legacy.sh", examples.TextOutput, nil).
-		WaitForPods(examples.NewMultiPodFetch("foo")).
-		WaitForPods(examples.NewMultiPodFetch("bar")).
-		WaitForPods(examples.NewMultiPodFetch("legacy")).
-		RunScript("curl-foo-bar-legacy.sh", examples.TextOutput, examples.GetCurlVerifier([]string{"200", "200", "200"})).
-		RunScript("verify-initial-policies.sh", examples.TextOutput, validateInitialPolicies).
-		RunScript("verify-initial-destinationrules.sh", examples.TextOutput, validateInitialDestinationRules).
-		RunScript("configure-mtls-destinationrule.sh", examples.TextOutput, nil).
-		RunScript("curl-foo-bar-legacy_post_dr.sh", examples.TextOutput, examples.GetCurlVerifier([]string{"200", "200", "200"})).
-		RunScript("httpbin-foo-mtls-only.sh", examples.TextOutput, nil).
-		RunScript("curl-foo-bar-legacy_httpbin_foo_mtls.sh", examples.TextOutput,
-			examples.GetCurlVerifier([]string{"200", "200", "000", "command terminated with exit code 56"})).
-		RunScript("cleanup.sh", examples.TextOutput, nil).
-		Run()
+	framework.
+		NewTest(t).
```

Comment:
Might be helpful for test debug purposes to continue supporting a name for a test.

Reply:
The name comes from the name of the test method (i.e. go's test infra) ... we're free to use whatever we want.